File: OJTrain/Python/leetcode1.py
from typing import List


class Solution:
    def __init__(self):
        return

    def twoSum(self, nums: List[int], target: int) -> List[int]:
        size = len(nums)
        for i in range(0, size):
            for j in range(i + 1, size):
                if nums[i] + nums[j] == target:
                    return [i, j]


solution = Solution()
print(solution.twoSum(nums=[2, 7, 11, 15], target=9))

File: OJTrain/Java/src/leetcode/LeetCode1.java
package leetcode;

public class LeetCode1 {
    public int[] twoSum(int[] nums, int target) {
        int i = 0, j = 1;
        boolean flag = true;
        for (i = 0; i < nums.length - 1; i++) {
            for (j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    flag = false;
                    break;
                }
            }
            if (!flag) {
                break;
            }
        }
        return new int[]{i, j};
    }
}
